modules/dispatcher/doc/dispatcher_admin.xml

```diff
@@ -1607,14 +1607,23 @@ opensips-cli -x mi ds_list
 		<itemizedlist>
 			<listitem><para>
 				<emphasis>partition</emphasis> (optional) - name of
-				the partition to be reloaded.
+				the partition to be reloaded. default partition is partition.
 			</para></listitem>
+
+			<listitem><para><emphasis>inherit_state</emphasis> (optional) : whether inherit old state of the destination , default is y. you must set partition, and then set inherit_state. </para>
```

Comment:
Actually it is not necessary to provide the name of the partition when using this new param. To avoid ambiguity of multiple optional params, you can use named params like this:
```
opensips-cli -x mi ds_reload inherit_state=n
```